.ci/.jenkins_exclude.yml

```diff
@@ -1,4 +1,27 @@
 exclude:
+  # Ruby 2.3
+  # Only includes rails-5.2, sinatra-2.0
```

Comment:
thanks for this comment, it's hard to see what _is_ included

Reply:
Definitely. This format is very hard to wrap ones head around.